### ISSUE TITLE
The result is strange, any help would be appreciated

### ISSUE THREAD

**Issue by HqChen2021** [sungho-CoolG/FSCL#4]
Hi, authors,

Thanks for sharing your code. When I tried to reproduce the reported result, something strange happened. I followed the instruction and ran the experiment on the CelebA dataset. The network architecture, optimizer setting, and dataset were all kept the same as the default setting of the provided code and as indicated in the Readme file. 

However, after running the `main_contrastive.py` for 100 epochs, the loss decreased from around 5 to around 3. (I wonder what loss value you got in your experiment?)  And I continued to run `main_classifier.py`  for ten epochs, and here, something strange happened.

1. The test accuracy was way too low than the reported result. I only got about 50% accuracy (using the pre-trained encoder,  `ckpt_epoch_100.pth`)
2. I switched to load the pre-trained model at the 10th epoch, i.e., `ckpt_epoch_10.pth`. It seems odd to me that the testing accuracy is still around 50%

If the reason for the  50% testing accuracy is that the encoder is not well-trained, why use a pre-trained encoder at the 100th epoch ( `ckpt_epoch_100.pth`) and the 10th epoch ( `ckpt_epoch_10.pth`) produce the similar testing accuracy? Should the former outperform the latter? Any suggestion would be appreciated.

**Comment by Omeiko** [comment]
@HqChen2021 I'm having the same issue. Have you managed to resolve it?
